src/NuGet/Microsoft.Orleans.Client.nuspec

```diff
@@ -18,9 +18,9 @@
     <copyright>Copyright Microsoft 2015</copyright>
     <tags>Orleans Cloud-Computing Actor-Model Actors Distributed-Systems C# .NET</tags>
     <dependencies>
-      <dependency id="Microsoft.Orleans.Core" version="1.0.3" />
-      <dependency id="Microsoft.Orleans.OrleansAzureUtils" version="1.0.3" />
-      <dependency id="Microsoft.Orleans.OrleansProviders" version="1.0.3" />
+      <dependency id="Microsoft.Orleans.Core" version="$version$" />
```

Comment:
OK, i got the first part.

I don't think keeping the Development package is viable though, for reasons i just posted below.